Hyperbolic_surface_triangulation_2/demo/Hyperbolic_surface_triangulation_2_Demo/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.1...3.15)
+
+project( demo )
+
+# Find includes in corresponding build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+include_directories(${CMAKE_BINARY_DIR})
+
+# Instruct CMake to run moc automatically when needed.
+set(CMAKE_AUTOMOC ON)
+
+
+# CGAL and its components
+find_package(CGAL REQUIRED COMPONENTS Core Qt5)
```

Comment:
We are at Qt6.  The transition is rather smooth

Reply:
done